src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1101,6 +1102,11 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
         cmd_helm_upgrade.extend(["--set", "global.isCustomCert={}".format(True)])
     if container_log_path is not None:
         cmd_helm_upgrade.extend(["--set", "systemDefaultValues.fluent-bit.containerLogPath={}".format(container_log_path)])
+    if enable_oidc_issuer:
+        if not auto_upgrade:
```

Comment:
this is not the right way to check if a cluster has auto_upgrade enabled. you would have to read this from the configmap and accordingly raise this warning